tests/enterprise_plugins/verify_tools.py

```
@@ -0,0 +1,68 @@
+                    suite.print_badge("Function tool_schema: INVALID RETURN TYPE", False)
+                    success = False
+                else:
+                    tool_names = {item.get("name") for item in schema if isinstance(item, dict)}
+                    required = {"bash_exec", "file_read", "file_write"}
+                    missing = sorted(required - tool_names)
+                    if missing:
+                        suite.print_badge(f"Function tool_schema: MISSING TOOLS {missing}", False)
+                        success = False
+                    else:
+                        suite.print_badge("Function tool_schema: VALID SCHEMA", True)
+            except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)